Cargo.toml

```
@@ -23,9 +23,12 @@ isomdl = { git = "https://github.com/spruceid/isomdl", rev = "1f4f762" }
 thiserror = "1.0.56"
 hex = "0.4.3"
 p256 = { version = "0.13.2", features = ["pkcs8"] }
+serde = "1.0.204"
 serde_cbor = "0.11.2"
 serde_json = "1.0.111"
+serde_derive = "1.0.204"
```

To reduce dependency complexity, use the `derive` feature from `serde` instead of depending on `serde_derive`.
```suggestion
serde = { version = "1.0.204", features = ["derive"] }
serde_cbor = "0.11.2"
serde_json = "1.0.111"
```

---

Fixed, will commit shortly.